Add docs

ahgnaw opened the pull request:
Use sphinx to generate documentation pages in a separate directory 'empyrical-docs'.

abhijeetkalyan commented:
I don't know enough about Sphinx to review this - @richafrank or @llllllllll might be better suited here?

llllllllll commented on README.rst:
```diff
@@ -0,0 +1,15 @@
+
+Installation
+-----------------
+::
```
`.. code-block:: bash`

llllllllll commented on docs/source/index.rst:
```diff
@@ -0,0 +1,23 @@
+.. empyrical documentation master file, created by
```
why not include the readme in the index instead of its own page?

llllllllll commented on empyrical/stats.py:
```diff
@@ -47,22 +47,18 @@ def annualization_factor(period, annualization):
     Parameters
     ----------
-    period : str, optional
+    :param period:
```
why did you change the docstrings from numpy style?

llllllllll commented:
can we use the readthedocs theme, we use it for zipline and I think it looks nicer than the default

llllllllll commented:
we also need intersphinx links for third party libs, I would look at the sphinx conf for zipline

llllllllll commented:
#12

ahgnaw commented:
@llllllllll Sure, i have no theme preferences.

I've gone ahead an added intersphinx links, changed the readme

I changed the docstring style because the docstrings were not compiling correctly. The documents produced looked wonky.

llllllllll commented:
Why is the build output dir outside of the repo, it is normally just `docs/build`

twiecki commented:
Is this PR still active? Should we merge it?

rsheftel commented:
Is this PR active? I have setup sphinx documentation on my fork that creates .rst files that could be used to automatically update the documentation if we hosted on ReadTheDocs. I am happy to create a PR for that if setting up ReadTheDocs something the package maintainers would like to do. Let me know.

twiecki commented:
that'd be great!

On Jan 28, 2018 9:17 PM, "rsheftel" <notifications@github.com> wrote:

> Is this PR active? I have setup sphinx documentation on my fork that
> creates .rst files that could be used to automatically update the
> documentation if we hosted on ReadTheDocs. I am happy to create a PR for
> that if setting up ReadTheDocs something the package maintainers would like
> to do. Let me know.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/quantopian/empyrical/pull/22#issuecomment-361092324>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AApJmKFCA03mstnZBZguXvEMZC94GF6Nks5tPNXqgaJpZM4JsUNy>
> .
>

rsheftel commented:
New PR #75 opened to replace this PR

rsheftel commented:
Now that #75 has been merged this PR is not needed and can be closed
